schemas/iso19139.che/src/main/plugin/iso19139.che/index-fields/default.xsl

```diff
@@ -776,6 +776,11 @@
     <xsl:for-each select="gmd:distributionInfo/gmd:MD_Distribution">
       <xsl:for-each select="gmd:distributionFormat/gmd:MD_Format/gmd:name/gco:CharacterString">
         <Field name="format" string="{string(.)}" store="true" index="true"/>
+        <Field name="formatWithVersion" string="{normalize-space(.)} ({normalize-space(../../gmd:version)})" store="true" index="true"/>
```

Comment:
maybe we should only put "(version)" only if there is one as version is missing quite often ?

Reply:
well after it's messy i think if several format in the same metadata, we'll have 2 arrays, hopefully format index will be same version index in those arrays but i don't really like it.
What is the problem with this index field ?

Reply:
Instead of adding a new field, maybe simply add this to the existing one ?

```
         <Field name="format" string="{if (../../gmd:version/gco:CharacterString != '') then concat(normalize-space(.) , ' (', normalize-space(../../gmd:version/*), ')') else normalize-space(.)}" store="true" index="true"/>
```

Same in GN.